apps/wallet/tailwind.config.js

```diff
@@ -28,6 +28,12 @@ module.exports = {
             width: {
                 7.5: '30px',
             },
+            borderRadius: {
+                '2lg': '0.625rem',
```

Comment:
Not a blocking concern but I don't love this naming. Probably should revamp all border radiuses with abhi.